q2_emperor/plugin_setup.py

```diff
@@ -30,6 +30,12 @@
     )
 }
 
+PLOT_PARAMETERS_DESC = PARAMETERS_DESC.copy().update({
+    'ignore_pcoa_features': 'By default biplot arrows cannot be visualized '
+                            'using this method. If you want to visualize '
+                            'biplot arrows use the "biplot" method'
+})
```

Comment:
`update` mutates in-place, this is ultimately setting `PLOT_PARAMETERS_DESC` to `None`.

Here is the help text:

![Screen Shot 2020-11-23 at 11 35 16 AM](https://user-images.githubusercontent.com/274668/100001177-f9cd2600-2d7f-11eb-80fa-403fbb0aaf86.png)

One solution (also note a small grammatical tweak proposed):

```suggestion
PLOT_PARAMETERS_DESC = {**PARAMETERS_DESC,
    'ignore_pcoa_features': 'Biplot arrows cannot be visualized using this '
                            'method. If you want to visualize biplot arrows '
                            'use the `biplot` method. Enabling this setting '
                            'will ignore any PCoA features that are present, '
                            'otherwise, if PCoA features are detected an '
                            'error will be raised.'
}
```